mmdet/models/backbones/res2net.py

```diff
@@ -335,9 +306,9 @@ class Res2Net(ResNet):
     """Res2Net backbone.
 
     Args:
-        depth (int): Depth of res2net, from {50, 101, 152}.
         scale (int): Scales used in Res2Net, normally 4.
-        baseWidth (int): basic width of each scale, normally 26.
+        base_width (int): basic width of each scale, normally 26.
```

Comment:
`Basic width of each scale. Default: 26`

Reply:
Fixed as suggested.